core/examples/README.md

```diff
@@ -13,3 +13,4 @@ See the particular subfolders for the installation instructions:
 * [Plain Javascript example application](/core/examples/luigi-example-js)
 * [Svelte example application](/core/examples/luigi-example-svelte)
 * [OpenUI5 example application](/core/examples/luigi-example-openui5)
+* [Next example application](/core/examples/luigi-example-next)
```

Comment:
```suggestion
* [NextJS example application](/core/examples/luigi-example-next)
```
Perhaps we should use NextJS, as it's the name used in the example and can't be confused with the word "next"